### PR TITLE
Updating the last few commands to remove EOF

### LINKED ISSUE #3813: Split kubectl commands using <<EOF into two steps
The current method for documenting `kubectl` commands YAML using `kubectl apply -f - <<EOF...` does not work for Windows users.

To fix this issue, do the following:

- [x] Update style rule in the [Documenting YAML](https://dev-knative.netlify.app/development/help/contributor/style-guide/documenting-code/) section of the style guide. File location: https://github.com/knative/docs/blob/mkdocs/docs/help/contributor/style-guide/documenting-code.md

     The new steps should be similar to the following:

   >1. Create a YAML file...
   >     ```
   >     # YAML FILE CONTENTS
   >     ```
   > 2. Apply the YAML file...
   >     ```
   >     kubectl apply --filename <filename>.yaml
   >     ```

- [x] Search the knative/docs repo for kubectl commands that use the `<<EOF` format. Update these commands to follow the new style rules.

     - Update all commands in the docs
     - Optional: Update commands in `tutorial` and `blog` folders 

See this [Slack thread](https://knative.slack.com/archives/C9CV04DNJ/p1623783139041200) for the discussion.

### PR DESCRIPTION
**Pull request by snneji**
This is the final PR for cleaning up `EOF` format `kubectl` commands. 

Fixes #3813.

### CODE REVIEW

**Comment by snneji** [comment]
I made the issue https://github.com/knative/docs/issues/4089 to deal with the failing test.

File: docs/serving/samples/hello-world/helloworld-java-spring/README.md [hunk 1]
```diff
@@ -68,96 +68,94 @@ cd knative-docs/docs/serving/samples/hello-world/helloworld-java-spring
...
-       Ensure that the container image value
-       in `service.yaml` matches the container you built in the previous step. Apply
-       the configuration using `kubectl`:
-
-       ```bash
-       kubectl apply --filename service.yaml
-       ```
-
+    1. Create a new file named `service.yaml` and copy the following service definition
+    into the file.
```
Comment (abrennan89) [inline docs/serving/samples/hello-world/helloworld-java-spring/README.md:132]:
```suggestion
    into the file:
```

File: docs/serving/samples/hello-world/helloworld-java-spring/README.md [hunk 2]
```diff
@@ -68,96 +68,94 @@ cd knative-docs/docs/serving/samples/hello-world/helloworld-java-spring
...
+              containers:
+                - image: docker.io/<username>/helloworld-java-spring
+                  env:
+                    - name: TARGET
+                      value: "Spring Boot Sample v1"
+        ```
+        Where `<username>` is your Docker Hub username.
+
+        Ensure that the container image value in `service.yaml` matches the
+        container you built in the previous step.
```
Comment (abrennan89) [inline docs/serving/samples/hello-world/helloworld-java-spring/README.md:152]: Should this be a note or something? Just a suggestion :)

**Comment by abrennan89** [comment]
/lgtm
/hold

Adding a hold til the test thing is fixed, feel free to remove it whenever you want to merge

**Comment by snneji** [comment]
I decided to use the original curly brace formatting for variables to avoid the failing test. The variable `{username}` is used consistently throughout the page. We can fix the style issue after #4089 has been done.

**Comment by abrennan89** [comment]
/lgtm
Thanks @snneji !

**Pull request merged**